adobe/react-spectrum
# Clear input value on Command + Delete for NumberField/ColorField

Closes #1877 

## ✅ Pull Request Checklist:


## 📝 Test Instructions:

For components that use useFormattedTextField (i.e. NumberField and ColorField):

1. Type text into input
2. Press CMD + Delete
3. Verify input is empty and remains active

## 🧢 Your Project:
